packages/perf/src/index.ts

```diff
@@ -0,0 +1,172 @@
+import { logger } from '@libp2p/logger'
```

Comment:
Please can you add a `@packageDocumentation` block that describes how to use this module.

See [circuit-relay](https://github.com/libp2p/js-libp2p/blob/55e1230ac5dd65ba573c2da4a0e5eae2bac81ed2/packages/libp2p/src/circuit-relay/index.ts#L1-L35) for an example.  It gets rendered [like this](https://libp2p.github.io/js-libp2p/modules/libp2p.circuit_relay.html).